src/hb-common.cc

```diff
@@ -792,8 +792,8 @@ static bool
 parse_feature_value_postfix (const char **pp, const char *end, hb_feature_t *feature)
 {
   bool had_equal = parse_char (pp, end, '=');
-  bool had_value = parse_uint (pp, end, &feature->value) ||
-                   parse_bool (pp, end, &feature->value);
+  bool had_value = parse_uint (pp, end, (unsigned int*) &feature->value) ||
+                   parse_bool (pp, end, (unsigned int*) &feature->value);
```

Comment:
Another option would be to define a local variable of appropriate type, and assign that to `feature->value` later.  Then you wouldn't need duplicating `parse_uint` and `parse_bool` etc.